terraform/variables.tf

```diff
@@ -0,0 +1,65 @@
+variable "region" {
+  default = "us-west-1"
+}
+
+variable "account" {
+  default = ""
+}
+
+#########
+# Redis #
+#########
+
+variable "redis-instance" {
```

Comment:
I realize you are setting the actual values in you `tfvars` file but I recommend that you also set some sort of default value here as well.

Reply:
:+1: I should add a pre commit hook doing this